File: Quera/Sadetar.py
"""
--------------------------------------------------
Name    : Sadetar
Author  : Danial Khorasanizadeh
Course  : Quera
Date    : 24/Nov/2020
Link    : https://quera.ir/problemset/contest/3403/%D8%B3%D8%A4%D8%A7%D9%84-%D8%B3%D8%A7%D8%AF%D9%87-%D8%AA%D8%B1
--------------------------------------------------
"""
num = list()
sum = 0
product = 1
for i in range(4):
    num.append(int(input()))
    sum = sum + num[i]
    product = product*num[i]
min = max = num[0]
for i in range(1, 4):
    if(num[i] > max):
        max = num[i]
    if(num[i] < min):
        min = num[i]
print("Sum : {0}\nAverage : {1}\nProduct : {2}\nMAX : {3}\nMIN : {4}".format(
    sum, (sum/4), product, max, min))

File: Quera/Sadetar.c
/*
--------------------------------------------------
Name    : Sade tar
Author  : Danial Khorasanizadeh
Course  : Quera
Date    : 24/Nov/2020
Link    : https://quera.ir/problemset/contest/3403/%D8%B3%D8%A4%D8%A7%D9%84-%D8%B3%D8%A7%D8%AF%D9%87-%D8%AA%D8%B1
--------------------------------------------------
*/
#include <stdio.h>
int main(void)
{
    float num[4], sum = 0, product = 1, min, max;
    for (int i = 0; i < 4; i++)
    {
        scanf("%f", &min);
        num[i] = min;
        sum = sum + num[i];
        product = product * num[i];
    }
    min = num[0];
    max = num[0];
    for (int j = 1; j < 4; j++)
    {
        if (num[j] > max)
        {
            max = num[j];
        }
        else if (num[j] < min)
        {
            min = num[j];
        }
    }
    printf("Sum : %.6f\nAverage : %.6f\nProduct : %.6f\nMAX : %.6f\nMIN : %.6f", sum, sum / 4, product, max, min);
    return 0;
}